src/app.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const artistRouter = require('./routes/artist');
+
+const app = express();
+
+const APP_PORT = process.env.PORT || 4000;
+
+app.use(express.json());
+
+app.use('/artist', artistRouter);
+
+app.get('/', (req, res) => {
+  res.status(200).json('Hello World');
+});
+
+
+module.exports = app;
```

Comment:
Just a formatting suggestion. Generally blank lines are used to separate different pieces of functionality/groups of logic into more manageable chunks, which is particularly important with larger files:
```
const express = require('express');
const artistRouter = require('./routes/artist');

const app = express();
const APP_PORT = process.env.PORT || 4000;

app.use(express.json());
app.use('/artist', artistRouter);
app.get('/', (req, res) => {
  res.status(200).json('Hello World');
});

module.exports = app;
```